src/chargepoint/transaction/TransactionManager.cpp

```diff
@@ -309,6 +309,12 @@ bool TransactionManager::handleMessage(const ocpp::messages::RemoteStartTransact
             authorized = m_smart_charging_manager.installTxProfile(Connectors::CONNECTOR_ID_CHARGE_POINT, request.chargingProfile);
         }
     }
+    else
```

Comment:
Strictly speaking this else block is redundant. I added it so the case is explicitly handled and documented with a comment. 
Let me know what you think

Reply:
It's a good idea, thanks.